exporter/collector/logs.go

```diff
@@ -141,6 +161,17 @@ func (l *LogsExporter) PushLogs(ctx context.Context, ld pdata.Logs) error {
 	return nil
 }
 
+func (l logMapper) getLogName(log pdata.LogRecord) (string, error) {
+	logNameAttr, exists := log.Attributes().Get("com.google.logName")
+	if exists {
+		return logNameAttr.AsString(), nil
+	}
+	if len(l.cfg.LogConfig.DefaultLogName) > 0 {
+		return l.cfg.LogConfig.DefaultLogName, nil
+	}
+	return "", fmt.Errorf("no default log name set")
```

Comment:
```suggestion
	return "", fmt.Errorf("no log name provided.  Set the 'default_log_name' option, or add the 'com.google.logName' resource attribute to set a log name.")
```
Maybe provide users a bit more info here.

Reply:
updated, except changed to just `attribute` (I spent time trying to get the `resourceAttributes` processor to work until I realized they are different)

Reply:
sgtm